cli/QuickstepCli.cpp

```diff
@@ -361,10 +361,14 @@ int main(int argc, char* argv[]) {
       sizeof(poison_message),
       quickstep::kPoisonMessage);
 
-  bus.Send(main_thread_client_id,
-           address,
-           style,
-           std::move(poison_tagged_message));
+  tmb::MessageBus::SendStatus send_status =
+      bus.Send(main_thread_client_id,
+               address,
+               style,
+               std::move(poison_tagged_message));
+  if (send_status != tmb::MessageBus::SendStatus::kOK) {
```

Comment:
Yes, code readability and performance reasons.

Please refer to the `CHECK Macros` section in [How To Use Google Logging Library (glog)](https://google-glog.googlecode.com/svn/trunk/doc/glog.html).